src/views/queries/queryResponses/viewResponseToScope3.tsx

```diff
@@ -0,0 +1,93 @@
+import Html from '@kitajs/html'
+import { singleton } from 'tsyringe'
+import { UUID } from '../../../models/strings.js'
+import { LinkButton, Page } from '../../common.js'
+type QueryStatus = 'resolved' | 'pending_your_input' | 'pending_their_input' | 'errored'
+
+interface Query {
+  id: UUID
+  company_name: string
+  query_type: string
+  updated_at: Date
+  status: QueryStatus
+  role: 'responder' | 'requester'
+  quantity: string
+  productId: string
+  emissions: string
+}
+@singleton()
+export default class Scope3CarbonConsumptionViewResponseTemplates {
+  //should be renamed to something more readable?
+  constructor() {}
+  public scope3CarbonConsumptionViewResponsePage = (query: Query) => {
+    return (
+      <Page
+        title="Veritable - Query Response"
+        activePage="queries"
+        heading="View response to your query"
+        headerLinks={[
+          { name: 'Queries', url: '/queries' },
+          { name: 'Query Request-someIdentifier', url: '/queries/query-response' },
```

Comment:
maybe use a varriable instead of 'someIdentifier'? e.g. company name

Reply:
Company name sounds like a good Idea thank you